src/overlays/actors/ovl_En_Sth/z_en_sth.c

```diff
@@ -350,9 +350,9 @@ s32 EnSth_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* p
     }
 
     if ((limbIndex == 8) || (limbIndex == 10) || (limbIndex == 13)) {
-        temp_v1 = limbIndex * 0x32;
-        rot->y += (Math_SinS(play->state.frames * (temp_v1 + 0x814)) * 200.0f);
-        rot->z += (Math_CosS(play->state.frames * (temp_v1 + 0x940)) * 200.0f);
+        overridePerLimb = limbIndex * FIDGET_MUL_I;
+        rot->y += Math_SinS(play->state.frames * (overridePerLimb + FIDGET_ADD_Y)) * FIDGET_SCALE;
+        rot->z += Math_CosS(play->state.frames * (overridePerLimb + FIDGET_ADD_Z)) * FIDGET_SCALE;
```

Comment:
I would like to have some more helpful names for `FIDGET_MUL_I`, `FIDGET_ADD_Y`, `FIDGET_ADD_Z`, and `overridePerLimb `

For the macros, instead of being literal and saying the operation like "add" or "mul", Id like if we can find a name that actually describes its purpose. Would you describe "add" as an offset? What is it achieving for the calculation? same with `MUL_I`.

`overridePerLimb` should probably not be named after "override", it should likely be named after fidgeting/how the fidget relates to the angle of the joints. But I still dont follow this calculation closely enough to give a proper suggestion.

Reply:
well, why not. fidgetting is a periodic sine/cosine function, so
```
FIDGET_PHASE_Y
FIDGET_PHASE_Z
FIDGET_FREQUENCY
```
seems to be a proper name